remo/featuredrep/templates/featuredrep_alter.jinja

```diff
@@ -33,6 +33,11 @@
         </span>
         {{ field_errors(form.users) }}
       </div>
+      <div class="large-6 columns">
+        <input type="text" class="flat" value="{{ form.created_on.value() }}"
+               name="created_on" placeholder="YYYY-MM-DD HH:MM:SS" />
```

Comment:
Thanks for the fixes. The layout looks better now. This placeholder here doesn't work though, as it shows "None" for me. I think that's the return value of form.created_on.value().